axonserver/src/main/java/io/axoniq/axonserver/message/command/CommandDispatcher.java

```diff
@@ -110,24 +111,38 @@ private void dispatchToCommandHandler(SerializedCommand command, CommandHandler
                                           Consumer<SerializedCommandResponse> responseObserver,
                                           ErrorCode noHandlerErrorCode, String noHandlerMessage) {
         if (commandHandler == null) {
-            logger.warn("No Handler for command: {}", command.getName() );
+            logger.warn("No Handler for command: {}", command.getName());
             responseObserver.accept(new SerializedCommandResponse(CommandResponse.newBuilder()
-                                                   .setMessageIdentifier(command.getMessageIdentifier())
-                                                   .setRequestIdentifier(command.getMessageIdentifier())
-                                                   .setErrorCode(noHandlerErrorCode.getCode())
-                                                   .setErrorMessage(ErrorMessageFactory.build(noHandlerMessage))
-                                                   .build()));
+                                                                                 .setMessageIdentifier(command.getMessageIdentifier())
+                                                                                 .setRequestIdentifier(command.getMessageIdentifier())
+                                                                                 .setErrorCode(noHandlerErrorCode
+                                                                                                       .getCode())
+                                                                                 .setErrorMessage(ErrorMessageFactory
+                                                                                                          .build(noHandlerMessage))
+                                                                                 .build()));
             return;
         }
 
-        logger.debug("Dispatch {} to: {}", command.getName(), commandHandler.getClient());
-        commandCache.put(command.getMessageIdentifier(), new CommandInformation(command.getName(),
-                                                                                command.wrapped().getClientId(),
-                                                                                responseObserver,
-                                                                                commandHandler.getClient(),
-                                                                                commandHandler.getComponentName()));
-        WrappedCommand wrappedCommand = new WrappedCommand(commandHandler.getClient(), command);
-        commandQueues.put(commandHandler.queueName(), wrappedCommand, wrappedCommand.priority());
+        try {
+            logger.debug("Dispatch {} to: {}", command.getName(), commandHandler.getClient());
+            commandCache.put(command.getMessageIdentifier(), new CommandInformation(command.getName(),
```

Comment:
putting an element in the cache may also fail. Note that the exception thrown is not a MessagingPlatformException, but a InsufficientCacheCapacityException.

Reply:
is the connecter capable to handle InsufficientCacheCapacityException? (not throwing error right away, instead of retrying few times)